src/main/java/com/avengers/gamera/controller/ArticleController.java

```diff
@@ -61,8 +61,16 @@ public void deleteLike(@PathVariable Long articleId) {
         likeService.deleteLike(articleId);
     }
 
-    @GetMapping("/{articleId}/likeNum")
+    @GetMapping("/{articleId}/like-num")
     public int getLikeNumForArticle(@PathVariable Long articleId) {
         return likeService.getLikeNumByArticleId(articleId);
     }
+
+    @GetMapping("/reviews/comment-num")
+    public PagingDto<List<MiniArticleGetDto>> getCommentNumForArticle(@RequestParam(defaultValue = "1") int page,
+                                                                      @RequestParam(defaultValue = "10") int size) {
```

Comment:
```suggestion
                                                                      @RequestParam(defaultValue = "根据前端UI来? 好像是5来着?") int size) {
```

 P.S. 如果不是为了练习 这里好像并不需要分页?

Reply:
改成5了哈